samples/core/src/test/java/io/oasp/gastronomy/restaurant/offermanagement/batch/impl/productimport/ProductManagementTest.java

```diff
@@ -1,9 +1,9 @@
 package io.oasp.gastronomy.restaurant.offermanagement.batch.impl.productimport;
 
 import static io.oasp.gastronomy.restaurant.offermanagement.common.OffermanagementTestDataConstants.ID_PRODUCT_MEAL_SCHNITZEL;
-import static io.oasp.gastronomy.restaurant.offermanagement.common.OffermanagementTestDataConstants.NUMBER_PRODUCT_DRINK;
-import static io.oasp.gastronomy.restaurant.offermanagement.common.OffermanagementTestDataConstants.NUMBER_PRODUCT_MEAL;
-import static io.oasp.gastronomy.restaurant.offermanagement.common.OffermanagementTestDataConstants.NUMBER_PRODUCT_SIDEDISH;
+import static io.oasp.gastronomy.restaurant.offermanagement.common.OffermanagementTestDataConstants.NUMBER_OF_PRODUCT_DRINK;
+import static io.oasp.gastronomy.restaurant.offermanagement.common.OffermanagementTestDataConstants.NUMBER_OF_PRODUCT_MEAL;
+import static io.oasp.gastronomy.restaurant.offermanagement.common.OffermanagementTestDataConstants.NUMBER_OF_PRODUCT_SIDEDISH;
 import static org.junit.Assert.assertEquals;

```

Comment:
I guess we need a rule for SonarQube and for Checkstyle/Eclipse that renderts a fat warning for ```org.junit.Assert```.